vm/systemSmartContracts/staking.go

```diff
@@ -205,6 +214,11 @@ func (r *stakingSC) changeRewardAddress(args *vmcommon.ContractCallInput) vmcomm
 
 	for i := 1; i < len(args.Arguments); i++ {
 		blsKey := args.Arguments[i]
+		err := r.eei.UseGas(r.gasCost.BaseOperationCost.DataCopyPerByte * uint64(len(blsKey)))
```

Comment:
the cost matches here, DataCopyPerByte both for changeValidatorKeys and changeRewardAddress

Reply:
SotrePerByte is used where the data is saved in the trie. CopyPerByte when the data is copied from one SC to another for input, than computed there in store if needed.

Reply:
ok